Monika After Story/game/script-topics.rpy

```diff
@@ -637,53 +634,43 @@ init python in mas_bookmarks_derand:
 #IN:
 #   initial_ask_text_multiple - Initial question Monika asks if there's multiple items to rerandom
 #   initial_ask_text_one - Initial text Monika says if there's only one item to rerandom
-#   talk_about_more_text - Question Monika asks if there's more things you'd like to rerandom
 #   caller_label - The label that called this label
 #   persist_var - The persistent variable which stores the derandomed eventlabels
 #   ev_db_code - The event database code for the topics we're rerandoming (Default: "EVE")
 label mas_rerandom:
     python:
         derandomlist = mas_get_player_derandoms(mas_bookmarks_derand.persist_var)
 
-        derandomlist.sort()
-        return_prompt_back = ("Nevermind.", False, False, False, 20)
+        derandomlist.sort(key=lambda _tuple: _tuple[0])
```

Comment:
tuples auto sort by the items they contain in index order, so this is redundant.
```suggestion
        derandomlist.sort()
```